data/budget-recommendations.csv

```diff
@@ -1,4231 +1,279 @@
-VND,SG,228920,676560
-VND,IL,199775,488665
-VND,KW,90110,439220
-VND,IT,135985,358995
-VND,JP,170500,484575
-VND,ES,139790,369670
-VND,VE,18325,283150
-VND,HU,117795,274075
-VND,PT,117590,306530
-VND,GR,144990,296945
-VND,KR,178765,473390
-VND,TW,145295,388965
-VND,PL,124320,307750
-VND,ZM,17955,102445
-VND,PR,43915,412820
-VND,CI,23965,99035
-VND,RO,114205,276330
-VND,BH,51860,287490
-VND,PA,34735,394605
-VND,GH,41630,116170
-VND,KH,24000,115255
-VND,SN,17475,125220
-VND,SA,74540,285405
-VND,RU,74075,236380
-VND,UG,17405,91510
-VND,BR,78980,194735
-VND,NI,17370,124690
-VND,MG,17575,88390
-VND,DO,21075,146550
-VND,MX,59560,215530
-VND,OM,37440,215540
-VND,CM,17345,160620
-VND,ZA,70970,228425
-VND,CL,73940,220970
-VND,KE,21345,92235
-VND,NP,17345,83780
-VND,PE,44165,185900
-VND,UA,42105,131500
-VND,MZ,17345,174435
-VND,TZ,2190,40030
-VND,AO,38465,100290
-VND,JO,28365,134700
-VND,ZW,17435,95590
-VND,TR,45180,123620
-VND,MU,23140,79070
-VND,SV,17345,95775
-VND,TH,67775,213265
-VND,CO,34375,133035
-VND,LK,16690,23055
-VND,CR,20955,139205
-VND,AR,46810,126790
-VND,MA,20905,82055
-VND,MY,59320,218145
-VND,KZ,25370,92865
-VND,DZ,20800,44030
-VND,ID,52930,196355
-VND,NG,29750,188200
-VND,GT,20845,147390
-VND,VN,43305,164120
-VND,BY,20335,79160
-VND,EG,27425,167505
-VND,UY,21620,99635
-VND,BD,17800,39405
-VND,PK,19515,112330
-VND,PH,33795,124710
-VND,ET,5420,41305
-VND,TN,20820,37725
-VND,LB,21020,62835
-VND,IN,19380,87540
-VND,PY,20830,66985
-VND,UZ,20845,40665
-VND,EC,20250,69470
-VND,MM,20850,22770
-VND,GE,20915,26650
-ZAR,US,330,935
-ZAR,CH,310,815
-ZAR,AU,280,720
-ZAR,GB,245,630
-ZAR,DE,245,620
-ZAR,DK,225,555
-ZAR,AT,165,510
-ZAR,NO,185,515
-ZAR,CA,205,630
-ZAR,NL,220,555
-ZAR,SE,175,475
-ZAR,IE,175,455
-ZAR,BE,160,470
-ZAR,FI,120,360
-ZAR,FR,150,395
-ZAR,NZ,150,440
-ZAR,HK,135,385
-ZAR,CZ,130,315
-ZAR,SK,125,265
-ZAR,AE,115,365
-ZAR,SG,150,445
-ZAR,IL,130,325
-ZAR,KW,60,290
-ZAR,IT,90,235
-ZAR,JP,115,320
-ZAR,ES,90,245
-ZAR,VE,10,185
-ZAR,HU,80,180
-ZAR,PT,80,205
-ZAR,GR,95,195
-ZAR,KR,120,315
-ZAR,TW,95,255
-ZAR,PL,80,205
-ZAR,ZM,10,70
-ZAR,PR,30,275
-ZAR,CI,15,65
-ZAR,RO,75,185
-ZAR,BH,35,190
-ZAR,PA,25,260
-ZAR,GH,30,75
-ZAR,KH,15,75
-ZAR,SN,10,85
-ZAR,SA,50,190
-ZAR,RU,50,155
-ZAR,UG,10,60
-ZAR,BR,50,130
-ZAR,NI,10,80
-ZAR,MG,10,60
-ZAR,DO,15,95
-ZAR,MX,40,145
-ZAR,OM,25,145
-ZAR,CM,10,105
-ZAR,ZA,45,150
-ZAR,CL,50,145
-ZAR,KE,15,60
-ZAR,NP,10,55
-ZAR,PE,30,125
-ZAR,UA,30,85
-ZAR,MZ,10,115
-ZAR,TZ,5,25
-ZAR,AO,25,65
-ZAR,JO,20,90
-ZAR,ZW,10,65
-ZAR,TR,30,80
-ZAR,MU,15,50
-ZAR,SV,10,65
-ZAR,TH,45,140
-ZAR,CO,25,90
-ZAR,LK,10,15
-ZAR,CR,15,90
-ZAR,AR,30,85
-ZAR,MA,15,55
-ZAR,MY,40,145
-ZAR,KZ,15,60
-ZAR,DZ,15,30
-ZAR,ID,35,130
-ZAR,NG,20,125
-ZAR,GT,15,95
-ZAR,VN,30,110
-ZAR,BY,15,50
-ZAR,EG,20,110
-ZAR,UY,15,65
-ZAR,BD,10,25
-ZAR,PK,15,75
-ZAR,PH,20,80
-ZAR,ET,5,25
-ZAR,TN,15,25
-ZAR,LB,15,40
-ZAR,IN,15,60
-ZAR,PY,15,45
-ZAR,UZ,15,25
-ZAR,EC,15,45
-ZAR,MM,15,20
-ZAR,GE,15,20
\ No newline at end of file
+Country,Budget,Currency
```

Comment:
I see the updated CSV with the correct currency is now available, with the columns: `budget_local_currency` and `budget_usd`. Are you planning to remove one of the columns in the CSV, or will you be adjusting the logic here?